client/src/components/ChapterCard.tsx

```diff
@@ -130,14 +132,19 @@ export const ChapterCard: React.FC<ChapterCardProps> = ({ chapter }) => {
                 >
                   {name}
                 </Link>
-                <Text fontWeight={600} fontSize={['sm', 'md', 'lg']}>
-                  {isPast(new Date(start_at)) ? 'Running' : 'Upcoming'}
-                </Text>
                 {invite_only && (
-                  <Text fontWeight={600} fontSize={['sm', 'md', 'lg']}>
+                  <LockIcon
+                    mt="2"
+                    marginLeft="auto"
+                    marginRight="1"
+                    fontSize={['sm', 'md', 'lg']}
+                  >
                     Invite only
```

Comment:
Text doesn't seem to be used anywhere when wrapped with icon. What do you think about adding tooltip in places where icon appears on it own? That means _ChapterCard_ and `/events/{id}` page.